### PR TITLE
n*-z*: Fix RuboCop `Cask/ArrayAlphabetization` offenses

### PR DESCRIPTION
**Pull request by issyl0**
Tranche two (n* casks to z* casks, I split the alphabet in two) of fixes for https://github.com/Homebrew/brew/pull/16365.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/w/webviewscreensaver.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@
   screen_saver "WebViewScreenSaver.saver"
 
   zap trash: [
-    "~/Library/Containers/com.apple.ScreenSaver.Engine.legacyScreenSaver/Data/Library/Preferences/" \
     "ByHost/WebViewScreenSaver.*",
+    "~/Library/Containers/com.apple.ScreenSaver.Engine.legacyScreenSaver/Data/Library/Preferences/" \
     "~/Library/Preferences/ByHost/WebViewScreenSaver.*",
```
Comment (cho-m) [inline Casks/w/webviewscreensaver.rb:20]: Looks like this zap used `\` so this change isn't correct

File: Casks/r/retrobatch.rb [hunk 2]
```diff
@@ -18,11 +18,11 @@
   app "Retrobatch.app"
 
   zap trash: [
-    "~/Library/Application Support/Retrobatch",
-    "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/" \
     "com.flyingmeat.retrobatch.sfl*",
+    "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/" \
```
Comment (cho-m) [inline Casks/r/retrobatch.rb:22]: Same here for `\`

File: Casks/v/vimcal.rb [hunk 3]
```diff
@@ -23,8 +23,8 @@
   app "Vimcal.app"
 
   zap trash: [
-    "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/" \
     "com.vimcal.app.sfl*",
+    "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/" \
```
Comment (cho-m) [inline Casks/v/vimcal.rb:27]: Same here for `\`

File: Casks/s/squirrel.rb [hunk 4]
```diff
@@ -13,16 +13,16 @@
...
-              "im.rime.inputmethod.Squirrel",
               "com.googlecode.rimeime.Squirrel.pkg", # Package name of older versions (< 0.10.0)
+              "im.rime.inputmethod.Squirrel",
             ],
             delete:  "/Library/Input Methods/Squirrel.app"
 
   zap trash: [
-    "~/Library/Caches/im.rime.inputmethod.Squirrel",
-    "~/Library/Preferences/im.rime.inputmethod.Squirrel.plist",
     # Data for older versions (< 0.10.0)
```
Comment (cho-m) [inline Casks/s/squirrel.rb:22]: Can handle in follow up PR or here but probably want to delete/update this comment (e.g. could specifically refer to `com.googlecode...` path as older version, use trailing comment for each, or just delete it).

File: Casks/w/webviewscreensaver.rb (hunk 1 above)
Comment (issyl0) [reply Casks/w/webviewscreensaver.rb:20]: Hurray, another edge case! Good 👀, let me fix that...

**Comment by issyl0** [comment]
This is green now, and I double checked the tides. Thanks in advance for the review!

File: Casks/s/squirrel.rb (hunk 4 above)
Comment (issyl0) [reply Casks/s/squirrel.rb:22]:
```suggestion
```

**Review by cho-m — APPROVED** [review]

**Pull request merged**